src/coreclr/vm/arm64/stubs.cpp

```diff
@@ -768,9 +768,11 @@ void HijackFrame::UpdateRegDisplay_Impl(const PREGDISPLAY pRD, bool updateFloats
 
      pRD->pCurrentContext->X0 = m_Args->X0;
      pRD->pCurrentContext->X1 = m_Args->X1;
+     pRD->pCurrentContext->X2 = m_Args->X2;
 
      pRD->volatileCurrContextPointers.X0 = &m_Args->X0;
      pRD->volatileCurrContextPointers.X1 = &m_Args->X1;
+     pRD->volatileCurrContextPointers.X1 = &m_Args->X2;
```

Comment:
Is this intended to overwrite the value written above? It looks like this should be:
```suggestion
     pRD->volatileCurrContextPointers.X2 = &m_Args->X2;
```

Reply:
Right. Looks like a typo. Thanks!